katello/katello-service

```diff
@@ -50,7 +50,7 @@ end
 
 def service_exists?(service)
   upstart = File.exist?("/etc/init.d/#{service}")
-  systemd = `systemctl is-enabled #{service} 2>&1` && $?.success?
+  systemd = `systemctl cat #{service} 2>&1` && $?.success?
```

Comment:
If a service is disabled `systemctl is-enabled #{service}` will exit with 1. So enabling a disabled service would fail silently as it would think the service doesn't exist. `systemctl cat` seems like the best way to test if a service exists.